src/app/pages/student/details/tabs/student-details-hours/student-details-hours.component.ts

```diff
@@ -69,15 +69,19 @@ export class StudentDetailsHoursComponent implements OnInit {
       if (!data) {
         return;
       }
+      const onlyCustomizedSlots: TimeSlot[] = this.student.schedule.filter((slot) => slot.customized);
```

Comment:
getting only the "customized" time slots, which are an indicator of student-specific data